### PR TITLE
fix vulnerable dependencies

### PR DESCRIPTION
**Pull request by pravinpushkar**
# Description

Related to below vulnerability failure - 

https://github.com/dapr/cli/actions/runs/4201647149/jobs/7288859657

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -228,8 +228,6 @@ require (
 
 replace (
 	github.com/Azure/go-autorest => github.com/Azure/go-autorest v14.2.0+incompatible
-
-	github.com/docker/distribution => github.com/docker/distribution v0.0.0-20191216044856-a8371794149d
```
Comment (pravinpushkar) [inline go.mod:232]: Removing this replace. Looks like it is related to helm dependency that has been upgraded recently.
@artursouza @mukundansundar  Do you see any issue with this?

Comment (mukundansundar) [reply go.mod:232]: LGTM. 

@pravinpushkar Can you see if the docker client can be upgraded also? 
The replace in the line below.

**Review by mukundansundar — APPROVED** [review]

File: go.mod (hunk 1 above)
Comment (mukundansundar) [reply go.mod:232]: Can be done as a separate PR.

Comment (pravinpushkar) [reply go.mod:232]: I will check that..

**Pull request merged**